src/prezzemolo/graph.py

```diff
@@ -87,6 +88,39 @@ def breadth_first_search(self, start: Vertex[ValueType], end: Vertex[ValueType])
         # start and end are type checked inside _breadth_first_search()
         return self._breadth_first_search(start, end, vertex_2_parent=None)
 
+    def _dijkstra_search(
+        self, start: Vertex[ValueType], end: Optional[Vertex[ValueType]], vertex_2_parent: Optional[Dict[Vertex[ValueType], Optional[Vertex[ValueType]]]]
+    ) -> bool:
+        distances: Dict[Vertex[ValueType], float] = {v: float("inf") for v in self.__vertexes}
+        distances[start] = 0.0
+        priority_queue: List[Tuple[float, Vertex[ValueType]]] = [(0.0, start)]
```

Comment:
Why not use an actual [priority queue](https://docs.python.org/3/library/queue.html?highlight=priority%20queue#queue.PriorityQueue) rather than a list?